src/Dfc.ProviderPortal.Courses/Helpers/IDocumentQueryExtensions.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Azure.Documents.Linq;
+
+namespace Dfc.ProviderPortal.Courses.Helpers
+{
+    public static class IDocumentQueryExtensions
+    {
+        // source: https://stackoverflow.com/questions/39338131/documentclient-createdocumentquery-async/49322332#49322332
+        public static async Task<List<T>> ToListAsync<T>(this IDocumentQuery<T> queryable)
```

Comment:
Can't help but think that an IEnumerable<T> implementation of this might be more reusable...

```
public static async Task<IEnumerable<T>> AsEnumerableAsync(this IDocumentQuery<T> queryable)
{
    while (queryable.HasMoreResults)
    {
        foreach (var result in await queryable.ExecuteNextAsync<T>())
        {
            yield return result;
        }
    }
}
```

Usage: `Something().AsQueryableAsync().ToList()`;

Reply:
![Selection_20201022-01-e3de8e3de8e3de8](https://user-images.githubusercontent.com/19378/96850656-fb3cc300-144e-11eb-8cce-9cfe35d466b2.png)

denied, this project hasn't been upgraded yet

Reply:
Doh!